dotnet/example_code/DynamoDB/CreateTablesLoadData.cs

```diff
@@ -1,13 +1,13 @@
-// snippet-sourcedescription:[CreateTablesLoadData.cs demonstrates how to create a DynamoDB table ]
+// snippet-sourcedescription:[CreateTablesLoadData.cs demonstrates how to ]
```

Comment:
Please do not remove descriptions or change the exiting code samples

Reply:
to add all these files i had to use automation..i already explained to dave bishop that we will bulk import them then update the description.. note that the single file that is there right now was a test